end-to-end-tests/specs/screenshot.spec.js

```diff
@@ -1,76 +1,73 @@
 var assert = require('assert');
 var expect = require('chai').expect;
 var waitForOncoprint = require('./specUtils').waitForOncoprint;
+var assertScreenShotMatch = require('../lib/testUtils').assertScreenShotMatch;
 
 const CBIOPORTAL_URL = process.env.CBIOPORTAL_URL.replace(/\/$/, "");
 
-function ssAssert(result, message){
-    assert(result[0].isWithinMisMatchTolerance);
-};
-
 function runResultsTests(){
 
     it('render the oncoprint', function(){
         waitForOncoprint();
         browser.pause(2000);
         var res = browser.checkElement('#oncoprint');
-        ssAssert(res);
+        assertScree(res);
```

Comment:
shouldn't it be `assertScreenShotMatch`?

Reply:
hah. yes.